### PR TITLE
Initial Flutter Guide

### PR DESCRIPTION
**Pull request by ntindle**
The Initial Guide for using the Flutter SDK

## Changes

- Add a ton to the blog post

## API Updates
N/A
### New Features *(required)*
More of a blog

### Deprecations *(required)*
N/A

### Enhancements *(optional)*
N/A

## Checklist

- [ ] Unit tests
- [x] Documentation

## References *(optional)*
N/A

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

**Comment by ma-sonr** [comment]
Good enough to kickoff flutter guide

**Review by ma-sonr — APPROVED** [review]
Good start

File: blog/2022-09-07-flutter-guide/index.mdx [hunk 1]
```diff
@@ -1,20 +1,714 @@
 ---
 slug: first-app-in-motor-flutter
-title: Building an App with motor-flutter
-authors: [prad.snr/]
+title: Building an App with motor_flutter
```
Comment (ma-sonr) [inline blog/2022-09-07-flutter-guide/index.mdx:3]:
```suggestion
title: Building an App with motor-flutter
```

File: blog/2022-09-07-flutter-guide/index.mdx [hunk 2]
```diff
@@ -1,20 +1,714 @@
...
 :::tip
 
-Use the power of React to create interactive blog posts.
+You can check that Flutter successfully installed with Flutter doctor:
+
+```bash title="terminal"
+flutter doctor
+```
+
+If you get any errors, now is a great time to Google™️ for fixes.
```
Comment (ma-sonr) [inline blog/2022-09-07-flutter-guide/index.mdx:25]:
```suggestion
If you run into installation issues, we suggest checking out [stack overflow](https://stackoverflow.com/search?q=flutter+install)!
```

File: blog/2022-09-07-flutter-guide/index.mdx [hunk 3]
```diff
@@ -1,20 +1,714 @@
...
+  // ...
+  }
+  ```
+2. Initialize the `MotorFlutter` class in the `main` function using `Get`:
+
+  ```dart title="lib/main.dart"
+  Future<void> main() async {
+    // Don't worry about this line just yet
+    WidgetsFlutterBinding.ensureInitialized();
+    // handle the motor intialization response
```
Comment (ma-sonr) [inline blog/2022-09-07-flutter-guide/index.mdx:169]:
```suggestion
    // handle the motor initialization response
```

**Review by ma-sonr — APPROVED** [review]

File: blog/2022-09-07-flutter-guide/index.mdx [hunk 4]
```diff
@@ -1,20 +1,714 @@
...
+    void _handleCreateAccountResponse(CreateAccountResponse? response) {
+      if (response != null) {
+        print(response);
+        // TODO: Store the credentials in the secure storage
+        print(response.whoIs);
+        Get.off(const LoginPage(title: "Login here"));
+      }
+    }
+  
+    // The meat and potatos of the file is here.
```
Comment (ntindle) [inline blog/2022-09-07-flutter-guide/index.mdx:600]:
```suggestion
    // The meat and potatoes of the file are here.
```

**Review by ma-sonr — APPROVED** [review]
great

**Pull request merged**